src/test/java/proj/kedabra/billsnap/presentation/controllers/BillControllerIT.java

```diff
@@ -1274,6 +1274,25 @@ void shouldReturnErrorWhenEditBillDoesNotHaveReferencedItemWhenEditingBill() thr
         assertThat(error.getMessage()).isEqualTo(ErrorMessageEnum.ITEM_ID_DOES_NOT_EXIST.getMessage(Long.toString(nonExistentItem)));
     }
 
+    @Test
+    @DisplayName("Should return error if taxes are null")
+    void shouldReturnErrorIfTaxesAreNull() throws Exception {
```

Comment:
(question) why's it required to have empty list instead of null to represent no taxes? It's inconsistent with the `accountsList` in bill creation for example, where the `accountsList` can be either null or [] to represent no invited accounts during bill creation. So maybe the accountsList should also be NotNull in that case

Reply:
It's because of how we're mapping things. Initially it was null, and then things exploded and I had to add this constraint. In that regards, yes, accountsList should probably be NotNull

Reply:
Yeah okay, it's actually because of how the mappers are with trying to map List<Resource> vs List<String> and there's issues with it. I'll add the notNull on accountsList